src/dataflow/lib/iot.ts

```diff
@@ -188,13 +199,17 @@ export class IoT {
               units,
               value: "",
               lastUpdateTime: Date.now(),
+              missing: false,
               plug}));
+          } else {
+            hub.setHubChannelMissingState(id, false);
+            hub.setHubChannelTime(id, Date.now());
```

Comment:
Here and in line `#201` above and for each different channel we're going to use a different timestamp. Seems like we should set the timestamp once at the top of the function and then use that single timestamp throughout.

Reply:
Fixed.